openmc/deplete/integrators.py

```diff
@@ -240,10 +228,9 @@ def __call__(self, n_bos, rates, dt, source_rate, _i=None):
         -------
         proc_time : float
             Time spent in CRAM routines for all materials in [s]
-        n_list : list of list of numpy.ndarray
-            Concentrations at each of the intermediate points with
-            the final concentration as the last element
-        op_results : list of openmc.deplete.OperatorResult
+        n_end : list of numpy.ndarray
+            Concentrations at end of interval
+        op_result : openmc.deplete.OperatorResult
```

Comment:
I don't think we want `op_result` returned here (it's not actually returned now so we should remove it from the docs).